src/main/java/com/internet/shop/dao/ProductDaoImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.internet.shop.dao;
```

Comment:
```suggestion
package com.internet.shop.dao.impl;
```
let's keep implementations in sub-package(same with services)